src/main/kotlin/org/elm/workspace/ui/ElmWorkspaceConfigurable.kt

```diff
@@ -25,19 +26,25 @@ class ElmWorkspaceConfigurable(
 
     private val elmVersionLabel = JLabel()
     private val elmFormatLabel = JLabel()
+    private val elmFormatActivateOnSaveHookCheckbox = JCheckBox()
 
-    override fun createComponent(): JComponent =
-            panel {
-                row("Directory containing 'elm' binary:") { pathToToolchainField(CCFlags.pushX) }
-                row("Elm version:") { elmVersionLabel() }
-                row("elm-format:") { elmFormatLabel() }
-            }
+    override fun createComponent(): JComponent {
+        elmFormatActivateOnSaveHookCheckbox.addChangeListener { _ -> update() }
+
+        return panel {
+            row("Directory containing 'elm' binary:") { pathToToolchainField(CCFlags.pushX) }
+            row("Elm version:") { elmVersionLabel() }
+            row("elm-format:") { elmFormatLabel() }
+            row("activate format-on-save") { elmFormatActivateOnSaveHookCheckbox() }
```

Comment:
Capitalize the first letter. Or maybe rename to "Run elm-format when file saved"?